api/secrets.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
```

Comment:
As I mentioned to @stevvooe, I feel strongly about having this as part of the control API.

We have a very simple concept in place since day 1: Cluster state can only be altered through the Control API - that's it.